cmd/kubeadm/app/cmd/token.go

```diff
@@ -158,11 +158,11 @@ func NewCmdToken(out io.Writer, errW io.Writer) *cobra.Command {
 	tokenCmd.AddCommand(listCmd)
 
 	deleteCmd := &cobra.Command{
-		Use:                   "delete [token-value]",
+		Use:                   "delete token-value [token-value] ...",
```

Comment:
Maybe `"delete [token-value] ..."`

Reply:
delete [token value] ... would mean that delete without any parameters is a valid command.
I'll change it to "delete token-value ..." I think. Any objections?